src/applications/disability-benefits/2346/reducers/form2346Reducer.js

```diff
@@ -0,0 +1,102 @@
+import {
+  FETCH_ACCESSORIES_DATA_FAILURE,
+  FETCH_ACCESSORIES_DATA_SUCCESS,
+  FETCH_BATTERY_DATA_FAILURE,
+  FETCH_BATTERY_DATA_SUCCESS,
+  FETCH_VETERAN_ADDRESS_FAILURE,
+  FETCH_VETERAN_ADDRESS_SUCCESS,
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // FETCH_SOCKS_DATA_FAILURE,
+  // FETCH_SOCKS_DATA_SUCCESS,
+  UPDATE_DATA_FAILURE,
+  UPDATE_DATA_SUCCESS,
+  FETCH_VETERAN_INFORMATION,
+  FETCH_VETERAN_INFORMATION_FAILURE,
+} from '../constants';
+
+const initialState = {
+  dlcBatteryData: [],
+  // NOTE: Decision was to take socks out of MVP -@maharielrosario at 1/28/2020, 9:49:20 AM
+  // dlcSocksData: [],
+  dlcAccessoriesData: [],
+  formData: {},
+};
+
```

Comment:
You could really shore up your initial state here and remove all those ```|| ''```

Reply:
We have `formData`

Reply:
@bradl3yC Can you explain about shore up the `initialState`?

Reply:
@mojaray2k sure!  Basically if you have a data object like ```formData``` and within that object there is an address and user object with various key values.  Your initialState should mirror those key/values and have a default value set to them.  That way when your component mounts and it pulls those data pieces from redux, if they're not populated yet by some API call they still have values as opposed to being undefined.

Reply:
Not sure why this was marked as resolved, @mojaray2k could you consider implementing @bradl3yC's suggestion here? 🙂

Reply:
My bad @bradl3yC Totally missed following up on this. I added that into initial state.

Reply:
@kelsonic That is now done. The commit has been pushed.

Reply:
Awesome, thank you! 🎉

Reply:
Oops sorry is `dateOfBirth` going to be a string or should we set it as `undefined` at first? @mojaray2k

Reply:
@kelsonic No it's a string also.

Reply:
@kelsonic We are using `moment` to format the `dateOfBIrth` and it needs to be a string.

Reply:
Understood! So default value is a string for `dob` and over time it's remaining a string even as the reducer is updated 👍Thanks for confirming!